Makefile

```diff
@@ -86,3 +86,8 @@ db-clean:
 db-fetch-vuln-list:
 	mkdir -p cache/vuln-list
 	wget -qO - https://github.com/aquasecurity/vuln-list/archive/main.tar.gz | tar xz -C cache/vuln-list --strip-components=1
+
+.PHONY: trivy-db-data
```

Comment:
We probably should add a comment about the reason why we have vuln-list and trivy-db-data.